### ISSUE TITLE
Nova Launcher

### ISSUE THREAD

**Issue by dentaku65** [Pablito2020/android_device_bq_krillin#6]
Hi,
I've followed all the steps described here 
https://github.com/Pablito2020/android_device_bq_krillin/issues/1#issuecomment-302926557
and everything went smooth.

Anwy installing Nova Launcher the (Ubuntu) phone Aquaris 4.5 keeps in continue reboot :-/

Reinstalled everything btw thx for your great work!

**Comment by Pablito2020** [comment]
Can you take logcats? It would be awseome.
Anyway.. I don't think that it is a device problem.. But logcat could be interesting in that situation.

**Comment by dentaku65** [comment]
Well I can do it, but I'm not aware how to do it with the device that keeps rebooting. To fix the situation I had to reflash the phone... can be that trebuchet launcher has been a core component of lineage os?

**Comment by Pablito2020** [comment]
No, It isn't... Trebuchet is only a launcher.apk added as the "stock" launcher in lineage os.
You can take logcat via computer. Install nova launcher, launch it and then send me the logcat
It seems an strange bug because for now we're running the kernel as permissive (anyway with selinux enforcing this error shouldn't pass) but... maybe the logcat say me that some lib file is missing and I have to compile it from source or something like that.
But without logcat I can't do anything.
Cheers,
Pablito2020

**Comment by MagnusAlm** [comment]
Old issue, but probably related to #13 

Tested to install it on my phone with cm13 compiled with the workaround and didn't have any issues with it.
